forge-gui/res/cardsfolder/upcoming/youve_been_caught_stealing.txt

```diff
@@ -0,0 +1,12 @@
+Name:You've Been Caught Stealing
+ManaCost:1 R
+Types:Sorcery
+A:SP$ Charm | Choices$ DBPump,DBToken
```

Comment:
A:SP$ Charm | Choices$ DBPump,DBBlock
~~SVar:DBPump:DB$ PumpAll | ValidCards$ Creature | SubAbility$ OppBlock | SpellDescription$ Threaten the Merchant — Each creature blocks this turn if able.~~
SVar:DBBlock:DB$ Effect | StaticAbilities$ KWPump | SpellDescription$ Threaten the Merchant — Each creature blocks this turn if able.